src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -2356,23 +2356,41 @@ End Module]]>,
 
     <Fact()>
     Public Sub ParseWarningDirective_LineContinuation2()
-        Dim tree = ParseAndVerify(<![CDATA[#Enable Warning _ 'Comment]]>,
-            <errors>
-                <error id="30203" message="Identifier expected." start="16" end="17"/>
-            </errors>)
+        ' PROTOTYPE LanguageVersion.VisualBasic15_3 should be LanguageVersion.VisualBasic15_5
+        Dim tree = ParseAndVerify(code:=<![CDATA[#Enable Warning _ 'Comment]]>,
+                                  options:=New VisualBasicParseOptions(LanguageVersion.VisualBasic15_3),
+                                    <errors>
+                                        <error id="37306" message="Please use language version 15.5 or greater to use comments after line continuation character." start="24" end="25"/>
+                                    </errors>)
         tree.VerifyOccurrenceCount(SyntaxKind.EnableWarningDirectiveTrivia, 2)
 
         Dim root = tree.GetRoot()
-        Dim skippedTokens = root.DescendantNodes(descendIntoTrivia:=True).OfType(Of SkippedTokensTriviaSyntax).Single
-        Assert.Equal(SyntaxKind.BadToken, skippedTokens.DescendantTokens.Single.Kind)
+        Dim skippedTokens = root.DescendantNodes(descendIntoTrivia:=True).OfType(Of SkippedTokensTriviaSyntax).SingleOrDefault
+        Assert.Null(skippedTokens)
 
         Dim enableNode = DirectCast(root.GetFirstDirective(), EnableWarningDirectiveTriviaSyntax)
         Assert.Equal(SyntaxKind.EnableKeyword, enableNode.EnableKeyword.Kind)
         Assert.False(enableNode.EnableKeyword.IsMissing)
         Assert.Equal(SyntaxKind.WarningKeyword, enableNode.WarningKeyword.Kind)
         Assert.False(enableNode.WarningKeyword.IsMissing)
-        Assert.True(enableNode.ErrorCodes.Single.IsMissing)
```

Comment:
>Assert.True(enableNode.ErrorCodes.Single.IsMissing) [](start = 8, length = 51)

Rather than removing the assert, we should assert that ```enableNode.ErrorCodes``` is empty

Reply:
Done